p2d/tex_utilities.py

```diff
@@ -77,6 +85,7 @@ def generate_statement_tex(problem, tex_dir):
         'LEGEND': problem['statement']['legend'],
         'INPUT': problem['statement']['input'],
         'OUTPUT': problem['statement']['output'],
```

Comment:
Also INPUT and OUTPUT may be None, right? If that is the case, treat them like you treat INTERACTION

Reply:
From what I've seen in polygon packages, they can only be a (possibly empty) string. But a sanity check for null is not a bad idea, just in case.